demo/src/app/components/shared/api-docs/api-docs.component.html

```diff
@@ -58,9 +64,16 @@ <h4>Inputs</h4>
       <h4>Outputs</h4>
       <table class="table table-sm">
         <tbody>
-          <tr *ngFor="let output of apiDocs.outputs">
-            <td class="label-cell"><code>{{output.name}}</code></td>
-            <td class="content-cell">{{output.description}}</td>
+          <tr *ngFor="let output of apiDocs.outputs" [class.deprecated]="output.deprecated">
+            <td class="label-cell">
+              <code>{{output.name}}</code><br>
+              <span *ngIf="output.since" class="badge badge-info">since {{ output.since.version }}</span>
+              <span *ngIf="output.deprecated" class="badge badge-secondary">deprecated {{ output.deprecated.version }}</span>
```

Comment:
nit: maybe it makes sense to create a component for the 2 above lines as it seems that the same thing is repeated in several places?